ansible/roles/baselayout/vars/main.yml

```diff
@@ -43,7 +43,7 @@ packages: {
   centos7_x64: ['git2u','centos-release-scl',], # centos-release-scl is required to enable SCLo
                                                # but we do it manually in partials/repo/centos7.yml for arm64
   centos7: [
-    'ccache,gcc-c++,devtoolset-6,sudo',
+    'ccache,gcc-c++,devtoolset-6,sudo,git,devtoolset-6-libatomic-devel',
```

Comment:
I've added `git` in #1765, if it gets merged first then you'll just have to resolve that conflict